pacaur

```diff
@@ -1103,9 +1103,70 @@ SearchInfoAur() {
     if [[ -z "$(grep -E "\-\-[r]?sort" <<< ${coweropts[@]})" ]]; then
         [[ $sortorder = descending ]] && coweropts+=("--rsort=$sortby") || coweropts+=("--sort=$sortby");
     fi
-    cower ${coweropts[@]} -- $@
-}
+    
+    # store cower information into an array, one line per package and values 
+    # separated by vertical slashes
+    formatopts='--listdelim # --format %n|%v|%d|%u|%p|%L|%P|%D|%M|%O|%C|%R|%m|%o|%r|%t|%s|%a\n'
+    readarray cower_array < <(cower ${formatopts} ${coweropts[@]} -- $@)
+
+    format_strings=( $"Name" $"Version" $"Description" $"URL" $"AUR Page" $"Licenses" $"Provides" $"Depends on" $"Make Deps" $"Optional Deps" $"Conflicts With" $"Replaces" $"Mantainer" $"Votes" $"Popularity" $"Out of Date" $"Submitted" $"Last Modified" ) 
+    
+    # in some languages more tabs are needed to properly adjust the info
+    format_tab=16
```

Comment:
How did you come with that number? If that value is different from one language to another one, it should be ideally retrieved dynamically.